File: 백준/Bronze/27433. 팩토리얼 2/팩토리얼 2.cc
#include <stdio.h>

long long fac(int num) {
	if (num <= 1) return 1;

	return num * fac(num - 1);
}

int main(void) {
	long long n = 0;
	if (scanf("%lld", &n) != 1) return 0;
	if (0 > n || n > 20) return 0;

	long long result = fac(n);

	printf("%lld\n", result);

	return 0;
}

File: 백준/Bronze/27433. 팩토리얼 2/팩토리얼 2.py
import sys

input = sys.stdin.readline
write = sys.stdout.write

def fac(num: int):
    if num <= 1: return 1
    
    return num * fac(num - 1)

def solve():
    n = int(input().rstrip())
    if 0 > n or n > 20: return
    
    result = fac(n)
    
    write(str(result) + "\n")
    
if __name__ == "__main__":
    solve()